components/nexperia/PMBT3904.stanza

```diff
@@ -28,62 +28,62 @@ public pcb-component component :
   symbol = sym(b => sym.b, c => sym.c, e => sym.e)
   property(self.rated-temperature) = min-max(-65.0, 150.0)
 
-  spice :
```

Comment:
Are you sure you want to merge those commented out blocks onto `dev` ?

Reply:
For now, I think so -- it does require a change to jitx-client to support it but I wanted to keep the code ready to go.